modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/reducers/fieldEditableReducer.es.js

```diff
@@ -105,6 +105,8 @@ function updateFieldAffectedByActivatingRepeatable({
 }) {
 	if (
 		field.type === 'date' &&
+		field.validation &&
+		field.validation.parameter &&
```

Comment:
You can use [Optional chaining (?.)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to simplify even more:
```suggestion
		field.validation?.parameter &&
```

Reply:
Hi @miltonmc thank you for your observations.
I force-pushed the recommended changes.